examples/provider/provider.tf

```diff
@@ -0,0 +1,4 @@
+provider "xray" {
+  url          = "${var.artifactory_url}/xray"
```

Comment:
can we not have `/xray` in the path unless `url_override = true`? It's an xray provider and this is redundant and very very rarely is it anything other than the default

Reply:
I don't see it's implemented in Artifactory or Projects. @alexhung should we do it sometime later?